### DISCUSSION TITLE
Pass arguments to pre scripts

### DISCUSSION THREAD

**Discussion by FedeRBeije** [vercel/turborepo#8963] [Help] [answered]
### Summary

Hy guys, i wonder how can i pass the arguments to my pre scripts.

This is the example:

in root i do `turbo run dev --filter=my-app -- -e example`

in my-app **package.json** i've a `dev` scripts and a `predev`scripts.

I need the argument `-e example` to be pass to the `predev` script, any suggestion if it possible?

right now if i do this only the direct script call 'dev' has access to it

### Additional information

I'm using turbo 2.0.11

**Comment by anthonyshew** [comment, marked as answer]
No, there is no way to selectively distribute passthrough flags today, as described [in the documentation](https://turbo.build/repo/docs/reference/run).
